src/en/juju-backups.md

```diff
@@ -0,0 +1,405 @@
+```
+
+Note that creating a backup may take a long time.
+
+
+### Managing Backups
+
+Each time a backup is created, it is also stored on the state-server for
+safekeeping, along with associated metadata giving it a unique id value and a 
+timestamp. This means that you can manage backups from whatever client you can
+connect from, and fetch previous backups if the originally downloaded file has
+gone astray. For managing backups, the `juju backups` command has an additional 
+set of subcommands:
+
+### juju backups download 
+
+usage: `juju backups download [options] <ID>`
+
+The `download` command will fetch the backup specified from the environment 
+and download it for local storage. If the `--filename` switch is not used, the
+backup is downloaded to the current directory and the filename used (derived 
+from a timestamp and the ID) is output to the console. 
+
+Examples:
+
+```bash
+juju backups download 20150914-150614.f41b1639-09b3-4eb7-8fbf-0927558cdf44
+```
+will return:
+
+```bash
+juju-backup-20150914-150614.f41b1639-09b3-4eb7-8fbf-0927558cdf44.tar.gz
+```
+
+
+### juju backups info  
+
+usage: `juju backups info [options] <ID>`
+  
+This command outputs the complete metadata record for the specified backup.
+For example:
+
+```bash
+juju backups info 20150914-150614.f41b1639-09b3-4eb7-8fbf-0927558cdf44
+```
+
+will result in output such as:
+
+```bash
+backup ID:       "20150914-150614.f41b1639-09b3-4eb7-8fbf-0927558cdf44"
+checksum:        "gECSwsOc2QtVG2RNrzk1bzCb+JU="
+checksum format: "SHA-1, base64 encoded"
+size (B):        32583286
+stored:          2015-09-14 15:06:30 +0000 UTC
+started:         2015-09-14 15:06:14.433210757 +0000 UTC
+finished:        2015-09-14 15:06:23.634473665 +0000 UTC
+notes:           ""
+environment ID:  "f41b1639-09b3-4eb7-8fbf-0927558cdf44"
+machine ID:      "0"
+created on host: "juju-f41b1639-09b3-4eb7-8fbf-0927558cdf44-machine-0"
+juju version:    1.24.5
+20150914-150614.f41b1639-09b3-4eb7-8fbf-0927558cdf44
+```
+
+
+### juju backups list    
+
+The `list` subcommand will display all the backups currently available on the 
+state server. There are two options for the output:
+
+```bash
+juju backups list
+```
+will display a complete list of all backup metadata for all the available 
+backups. As such, it can output a lot of data, so you may consider using 
+the `--brief` switch on the command, which will return just a list of the 
+backup ID values:
+
+```bash
+juju backups list --brief
+```
+
+should return something like:
+
+```bash
+20150910-150614.f41b1639-09b3-4eb7-8fbf-0927558cdf44
+20150911-192138.f41b1639-09b3-4eb7-8fbf-0927558cdf44
+20150912-100824.f41b1639-09b3-4eb7-8fbf-0927558cdf44
+20150914-160331.f41b1639-09b3-4eb7-8fbf-0927558cdf44
+```
+
+
+### juju backups remove  
+usage: `juju backups remove [options] <ID>`
+
+If you wish to remove a particular backup file from the state server (perhaps to
+save space!), you can use the remove subcommand with the appropriate ID:
```

Comment:
Should we be providing minimum requirements for state servers in terms of space required for backups?